docs/api/SystemAPI.md

```diff
@@ -0,0 +1,3877 @@
+<!-- Generated automatically, DO NOT EDIT! -->
```

Comment:
Why are we adding a new file (SystemAPI.md)? It needs to modify the existing file (https://github.com/rdkcentral/rdkservices/blob/main/docs/api/SystemPlugin.md)